src/git/objects/commits.js

```diff
@@ -47,8 +48,7 @@ export default (client, repo, store) => {
             commitMessage: message,
             commitAuthor: {
                 ...author,
-                // @TODO: Set new Date() with the good format
-                date: '2016-07-24T00:49:30+02:00',
+                date: moment().format(),
```

Comment:
There are barely different:
```js
(new Date()).toISOString() // 2016-08-25T15:46:19.126Z
moment().format()          // 2016-08-25T17:46:19+02:00
```

According to the [documentation](https://developer.github.com/v3/git/commits/#optional-parameters), Github API needs the second format.